machine_manager/README.rst

```diff
@@ -0,0 +1,27 @@
+Machine Manager
```

Comment:
Ok @Daniel-CA (forget vehicles) thanks for your work on this matter. But if i have network devices, machines and tools to manage with my ERP, I wish to have at least a common base module to manage them all and then a machine module for machine specific fields and behavior overrides. 
You could manage polymorphism with inheritS to deal with these specific fields like product.template and product.product and specific views.
OCA repo should avoid non generic modules when generic praticals are a common ways (main stream). 
Thanks you again for this module.